test/couch_mrview_all_docs_tests.erl

```diff
@@ -107,6 +107,21 @@ should_query_with_limit_and_skip(Db) ->
     ]},
     ?_assertEqual(Expect, Result).
 
+should_query_with_config_limit_and_skip(Db) ->
```

Comment:
(neither is `/Users/robertnewson/Source/asf/couchdb/src/couch_mrview/test/couch_mrview_map_views_tests.erl:97: Warning: function should_map_with_config_limit/1 is unused`)

This is cause we need to mention them explicitly above, they're not auto-detected